packages/jellyfish-api-core/src/category/token.ts

```diff
@@ -134,3 +147,13 @@ export interface TokenPagination {
   including_start: boolean
   limit: number
 }
+
+export interface CustomTxInfo {
+  type: string
+  valid: boolean
+  results: any
+  blockhash: string
+  blockHeight: number
+  blockTime: number
+  confirmations: number
+}
```

Comment:
<img width="508" alt="Screenshot 2021-05-15 at 11 16 51 PM" src="https://user-images.githubusercontent.com/7101806/118368984-3d894580-b5d5-11eb-9b46-89f0d0204076.png">
<img width="608" alt="Screenshot 2021-05-15 at 11 27 31 PM" src="https://user-images.githubusercontent.com/7101806/118368991-4417bd00-b5d5-11eb-8e6f-dade7d119787.png">

What was written in ain is wrong.
When I tried to console.log, there is blockTime